http-server/src/lib.rs

```diff
@@ -35,6 +35,9 @@ mod server;
 /// Common builders for RPC responses.
 pub mod response;
 
+/// Common tower middlewares exposed for RPC interaction.
+pub mod middlewares;
```

Comment:
```suggestion
pub mod middleware;
```
? :)

Reply:
It sounds better, have modified it! Thanks!